OpenRA.Mods.Common/ServerTraits/LobbyCommands.cs

```diff
@@ -186,8 +186,11 @@ public bool InterpretCommand(S server, Connection conn, Session.Client client, s
 								server.LobbyInfo.Clients.Remove(occupant);
 								server.SyncLobbyClients();
 								var ping = server.LobbyInfo.PingFromClient(occupant);
-								server.LobbyInfo.ClientPings.Remove(ping);
-								server.SyncClientPing();
+								if (ping != null)
+								{
+									server.LobbyInfo.ClientPings.Remove(ping);
```

Comment:
I wonder why Coverity complains there. `List<T>.Remove()` deals just fine with `null` arguments. Any `SyncClientPing` doesn't use it.